### PR TITLE
feat: add my contribution profile

### PR DESCRIPTION
**Pull request by suzy-g38**
## Added Contribution Profile
I added my contribution profile to the contribution folder.

### CODE REVIEW

File: contribution/suzy-g38.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+author: Sulagna Ghosh
+github_username: suzy-g38
+occupation:
```
Comment (priyankarpal) [inline contribution/suzy-g38.mdx:4]: you can add your `occupation`

File: contribution/suzy-g38.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          </svg>
+        }
+        level=""
```
Comment (priyankarpal) [inline contribution/suzy-g38.mdx:27]: add your level. such as `newbie`, `pro`, etc or you can remove the code.

File: contribution/suzy-g38.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+      >
+        
+      </Skill>
+      <Skill
+        name="NextJs"
+        href="https://nextjs.org/"
+        level=""
```
Comment (priyankarpal) [inline contribution/suzy-g38.mdx:34]: also here.

File: contribution/suzy-g38.mdx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+       
+      </Skill>
+    </Skills>
+  </Tab>
+  <Tab title="Tools">
+    <Skills title="Tools">
+      <Skill
+        name="VS Code"
+        href="https://code.visualstudio.com/"
+        level=""
```
Comment (priyankarpal) [inline contribution/suzy-g38.mdx:51]: also here.

**Review by priyankarpal — CHANGES_REQUESTED** [review]
Looks Good 🎉 but needs some small changes.

File: contribution/suzy-g38.mdx (hunk 4 above)
Comment (suzy-g38) [reply contribution/suzy-g38.mdx:51]: > also here.
okay

**Pull request closed without merging**